autogen.sh

```
@@ -66,18 +66,23 @@ args="$args $@"
 cd $oldpwd
 
 if [ "$verb" = "c" ]; then
+        set -x
         $topdir/configure CFLAGS='-g -O0 -ftrapv' $args
-        make clean
+        make clean >/dev/null
```

shouldn't this be `2> /dev/null`?

---

I don't know, should it? I'm not trying to silence errors, just the usual 'make clean' uninformative spam.

With the patch I have:

```
./autogen.sh c --disable-manpages
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
libtoolize: linking file 'build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: linking file 'm4/libtool.m4'
libtoolize: linking file 'm4/ltoptions.m4'
libtoolize: linking file 'm4/ltsugar.m4'
libtoolize: linking file 'm4/ltversion.m4'
libtoolize: linking file 'm4/lt~obsolete.m4'
configure.ac:33: installing 'build-aux/compile'
configure.ac:37: installing 'build-aux/missing'
Makefile.am: installing 'build-aux/depcomp'
+ ./configure 'CFLAGS=-g -O0 -ftrapv' --sysconfdir=/etc --localstatedir=/var --libdir=/usr/lib64 --disable-manpages
configure: loading site script /usr/share/config.site
checking for gcc... gcc
...
        LDFLAGS:   -Wl,--as-needed -Wl,--no-undefined -Wl,-z,relro -Wl,-z,now -pie -Wl,-fuse-ld=gold   

+ make clean
./autogen.sh c --disable-manpages  33.65s user 9.32s system 96% cpu 44.388 total
```

i.e. it seems to be working.